android/app/src/main/java/org/fossasia/openevent/dbutils/RealmDataRepository.java

```diff
@@ -357,6 +357,10 @@ public Session getSessionSync(int sessionId) {
         return realm.where(Session.class).equalTo("id", sessionId).findFirst();
     }
 
+    public RealmResults<Session> getSessions() {
```

Comment:
Can you explain why you needed to add this ? I mean how was the session fetched before implementing the ViewModel ?

Reply:
Like actually I wanted to fetch all microlocations , speakers , tracks and sessions so I created a method for Sessions and Speakers differently as they were not present before (I sorted them according to their title and name respectively). Then I added all of them to a an arraylist of object type.

Reply:
Actually my question is if it wasn't present before , then how was it implemented before ? I mean it was working before, it's just a refactor from View to ViewModel. Ideally there shouldn't be any changes to the repository layer.

Reply:
Actually Realm.where(Speaker.class) is used to find the full list. There wasn't a method before like it did in the case of microlocations and tracks 😅

Reply:
Yes, then please don't change the implementation. Implement it the way it was done before. Incase you want to change the implementation, create an issue for it.

Reply:
Please find the methods like ```addResultsFromLocations``` to get the idea of how to fetch only ```Locations``` from the database. I think you should first go through and understand it.